### PR TITLE
Fix for #320 #325 #323, fix propTypes across RNE

### PR DESCRIPTION
**Pull request by bozzmob**
This PR is fix PropTypes across RNE, so that the behavior uniform.

Related Issues:
#320 Warning sending color to searchBar
#323  Overuse of react-native's custom PropType View.propTypes.style
#325 Icon: invalid proptype color

Related PR:
#307 Fix propTypes 

Additional note: Lets please squash the commits into one commit (with title of the PR) and then merge.

### CODE REVIEW

**Comment by binoy14** [comment]
Let's think about how can we add tests to validate propTypes 

P.S Can you please squash

**Comment by bozzmob** [comment]
@binoy14 Please review the changes. Will squash it before merging.

File: docs/index.md [hunk 1]
```diff
@@ -13,10 +13,12 @@
 </p>
 
 <p align="center">
+  <a href="#backers"><img src="https://opencollective.com/react-native-elements/backers/badge.svg"></a>
+  <a href="#sponsors"><img src="https://opencollective.com/react-native-elements/sponsors/badge.svg"></a>
```
Comment (Monte9) [inline docs/index.md]: these changes exist on master. Make sure you rebase `master` on `next`

File: docs/index.md [hunk 2]
```diff
@@ -13,10 +13,12 @@
...
   <a href="https://www.npmjs.com/package/react-native-elements"><img src="https://img.shields.io/npm/v/react-native-elements.svg?style=flat-square"></a>
   <a href="https://www.npmjs.com/package/react-native-elements"><img src="https://img.shields.io/npm/dm/react-native-elements.svg?style=flat-square"></a>
   <a href="https://travis-ci.org/react-native-training/react-native-elements"><img src="https://img.shields.io/travis/react-native-training/react-native-elements/master.svg?style=flat-square"></a>
-  <a href="https://codecov.io/gh/react-native-training/react-native-elements"><img src="https://codecov.io/gh/react-native-training/react-native-elements/coverage.svg?branch=v0.11.0"></a>
+  <a href="https://codecov.io/gh/react-native-training/react-native-elements"><img src="https://codecov.io/gh/react-native-training/react-native-elements/coverage.svg"></a>
```
Comment (Monte9) [inline docs/index.md]: same here.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-elements",
-  "version": "0.11.1",
+  "version": "0.11.0",
```
Comment (Monte9) [inline package.json]: Rebase here too

File: src/buttons/ButtonGroup.js [hunk 4]
```diff
@@ -120,6 +132,7 @@ ButtonGroup.propTypes = {
   lastBorderStyle: PropTypes.oneOf(View.propTypes.style, NativeText.propTypes.style),
   buttonStyle: View.propTypes.style,
   selectedBackgroundColor: PropTypes.string,
+  containerBorderRadius: PropTypes.number,
```
Comment (Monte9) [inline src/buttons/ButtonGroup.js]: please make sure this is updated on the docs.

File: src/containers/Card.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes } from 'react';
-import { View, StyleSheet, Platform, Image } from 'react-native';
+import { View, StyleSheet, Platform, Image,Text as NativeText } from 'react-native';
```
Comment (Monte9) [inline src/containers/Card.js]: missing a space. Also won't have to worry about this int he future as we are adding prettier.

File: src/form/FormInput.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, { Component, PropTypes } from 'react';
-import { TextInput, StyleSheet, View, Platform, Dimensions } from 'react-native';
+import { TextInput, StyleSheet, View, Platform, Dimensions,Text as NativeText } from 'react-native';
```
Comment (Monte9) [inline src/form/FormInput.js]: space

File: src/form/FormLabel.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes } from 'react';
-import { StyleSheet, View, Platform } from 'react-native';
+import { StyleSheet, View, Platform,Text as NativeText } from 'react-native';
```
Comment (Monte9) [inline src/form/FormLabel.js]: space

**Comment by Monte9** [comment]
@bozzmob Changes look good to me. Please make sure once this PR's is merged that you comment on all the related issue about this merge and that the fix will be released in `v0.11.2`

File: src/form/FormLabel.js (hunk 7 above)
Comment (binoy14) [inline src/form/FormLabel.js]: Prettier will do it ;)

**Pull request merged**